Example/Sources/ConversationViewController.swift

```diff
@@ -309,16 +309,16 @@ extension ConversationViewController: MessagesDisplayDelegate {
         return annotationView
     }
 
-    func animationBlockForLocation(message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> ((UIImageView) -> Void)? {
-        return { view in
-            view.layer.transform = CATransform3DMakeScale(0, 0, 0)
-            view.alpha = 0.0
-            UIView.animate(withDuration: 0.6, delay: 0, usingSpringWithDamping: 0.9, initialSpringVelocity: 0, options: [], animations: {
-                view.layer.transform = CATransform3DIdentity
-                view.alpha = 1.0
-            }, completion: nil)
-        }
-    }
+//    func animationBlockForLocation(message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> ((UIImageView) -> Void)? {
+//        return { view in
+//            view.layer.transform = CATransform3DMakeScale(0, 0, 0)
+//            view.alpha = 0.0
+//            UIView.animate(withDuration: 0.6, delay: 0, usingSpringWithDamping: 0.9, initialSpringVelocity: 0, options: [], animations: {
+//                view.layer.transform = CATransform3DIdentity
+//                view.alpha = 1.0
+//            }, completion: nil)
+//        }
+//    }
```

Comment:
Is there a reason we want this commented out?

Reply:
Not sure 😅 It's in progress. I don't think I'm going to be able to remove the subclasses. It doesn't make sense to support multi text messages to be honest. If anything far down the road we can have a subclass for urls/file previews that shows both an image and text